funnel/models/venue.py

```diff
@@ -25,7 +32,9 @@ class Venue(UuidMixin, BaseScopedNameMixin, CoordinatesMixin, db.Model):
         grants_via={None: project_child_role_map},
     )
     parent = sa.orm.synonym('project')
-    description = MarkdownColumn('description', default='', nullable=False)
+    description = MarkdownColumnNative(  # type: ignore[has-type]
+        'description', profile='basic', default='', nullable=False
```

Comment:
Should `profile='basic'` be the default value?

Reply:
It's not a default value. This specifies `Venue.description` shall use the `basic` profile.